docs/presentation/Content/ContentRelatedInstances.md

```diff
@@ -1,38 +1,151 @@
-
-## Example
-
-```JSON
-{
-  "specType": "ContentRelatedInstances",
-  "relationshipPaths": [{
-    "relationship": { "schemaName": "BisCore", "className": "ModelContainsElements" },
-    "direction": "Forward",
-    "targetClass": { "schemaName": "BisCore", "className": "PhysicalElement" }
-  }],
-  "arePolymorphic": true
-}
+| Name                                                      | Required? | Type                                                                                  | Default |
+| --------------------------------------------------------- | --------- | ------------------------------------------------------------------------------------- | ------- |
+| *Filtering*                                               |
+| [`relationshipPaths`](#attribute-relationshippaths)       | Yes       | [`RelationshipPathSpecification[]`](../Common-Rules/RelationshipPathSpecification.md) |
+| [`instanceFilter`](#attribute-instancefilter)             | No        | [ECExpression](./ECExpressions.md#instance-filter)                                    | `""`    |
+| [`onlyIfNotHandled`](#attribute-onlyifnothandled)         | No        | boolean                                                                               | `false` |
+| *Ordering*                                                |
+| [`priority`](#attribute-priority)                         | No        | `number`                                                                              | `1000`  |
+| *Content Modifiers*                                       |
+| [`relatedProperties`](#attribute-relatedproperties)       | No        | `RelatedPropertiesSpecification[]`                                                    | `[]`    |
+| [`calculatedProperties`](#attribute-calculatedproperties) | No        | `CalculatedPropertiesSpecification[]`                                                 | `[]`    |
+| [`propertyCategories`](#attribute-propertycategories)     | No        | `PropertyCategorySpecification[]`                                                     | `[]`    |
+| [`propertyOverrides`](#attribute-propertyoverrides)       | No        | `PropertySpecification[]`                                                             | `[]`    |
+| [`showImages`](#attribute-showimages)                     | No        | `boolean`                                                                             | `false` |
+| *Misc.*                                                   |
+| [`relatedInstances`](#attribute-relatedinstances)         | No        | [`RelatedInstanceSpecification[]`](../Common-Rules/RelatedInstanceSpecification.md)   | `[]`    |
+
+### Attribute: `relationshipPaths`
+
+Specifies a list of [relationship path specifications](../Common-Rules/RelationshipPathSpecification.md) to follow when looking for related instances.
+Either a single or an array of specifications may be specified for this attribute. Specifying an empty array makes the whole specification a no-op.
```

Comment:
@grigasp Is my suggestion factually accurate?

```suggestion
Specifies a chain of [relationship path specifications](../Common-Rules/RelationshipPathSpecification.md) that forms a path from an input instance to the output instances. When this array is empty, the specification produces no results.
```

Reply:
yes